array-methods.js

```diff
@@ -39,7 +66,22 @@ var sumOfInterests = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var stateSums = null;
+
+var stateSums = {};
+
+dataArray.forEach(
+  function(i){
```

Comment:
The function declaration should be on the line above. Make sure the rest of the file follows this convention.